Tests/MathLib/TestCVode.cpp

```diff
@@ -121,7 +121,7 @@ TEST(MathLibCVodeTest, ZeoliteAdsorption)
 
 	ode_solver->setFunction(f_zeolite, nullptr);
 
-	ode_solver->setIC(0.0, {pV, C});
+	ode_solver->setIC(0.0, {{pV, C}});
```

Comment:
Can be. I do remember something about std::array. On the other hand std::array is just struct, so it could be related to struct's initialization.